sdk/tables/azure-tables/samples/javascript/src/shardKey.js

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I think there is a typo in the file name

Reply:
Good catch 😄